hybridse/src/udf/udf.cc

```diff
@@ -49,6 +49,19 @@ using hybridse::codec::Row;
 using openmldb::base::StringRef;
 using openmldb::base::Timestamp;
 using openmldb::base::Date;
+
+void hex(StringRef *str, StringRef *output) {
+    std::ostringstream ss;
+    for (uint32_t i=0; i < str->size_; i++) {
+        ss << std::hex << std::uppercase << int(str->data_[i]);
+    }
+    output->size_ = ss.str().size();
```

Comment:
since `ss.strr()` returns a copy of string, it is better to call it once and reuse the same string temporary variable